### DISCUSSION TITLE
how can i create a unique signal per app instance?

### DISCUSSION THREAD

**Discussion by meir-galilsoftware** [preactjs/signals#544] [Q&A]
hi :)
we want to create unique signals per instance of the app. the app can be installed couple of times in the same page, but the import of the signal in the App, does not generate 2 signals.

here is a demo of the problem.
i would appreciate any idea :)
thanks
 
https://stackblitz.com/edit/react-signal-for-state-management-sr6tqk-ztpfbv

**Comment by rschristian** [comment]
The simple answer is to just create two signals, or initialize them from the global one into the context API or something.
